molgenis-app-gavin2/src/main/java/org/molgenis/app/gavin/meta/GavinRun.java

```diff
@@ -0,0 +1,114 @@
+package org.molgenis.app.gavin.meta;
+
+import static org.molgenis.app.gavin.meta.GavinRunMetadata.DISCARDED_INPUT_FILE;
+import static org.molgenis.app.gavin.meta.GavinRunMetadata.FILTERED_INPUT_FILE;
+import static org.molgenis.app.gavin.meta.GavinRunMetadata.FINISHED_AT;
+import static org.molgenis.app.gavin.meta.GavinRunMetadata.ID;
+import static org.molgenis.app.gavin.meta.GavinRunMetadata.INPUT_FILE_NAME;
+import static org.molgenis.app.gavin.meta.GavinRunMetadata.OUTPUT_FILE;
+import static org.molgenis.app.gavin.meta.GavinRunMetadata.STARTED_AT;
+import static org.molgenis.app.gavin.meta.GavinRunMetadata.STATUS;
+import static org.molgenis.app.gavin.meta.GavinRunMetadata.SUBMITTED_AT;
+
+import java.time.Instant;
+import org.molgenis.data.Entity;
+import org.molgenis.data.file.model.FileMeta;
+import org.molgenis.data.meta.model.EntityType;
+import org.molgenis.data.support.StaticEntity;
+import org.molgenis.jobs.model.JobExecution.Status;
+
+public class GavinRun extends StaticEntity {
```

Comment:
Optional<String> instead of String for nullable attributes to prevent NPEs occuring downstream.